### PR TITLE
Support SMS

### PR DESCRIPTION
**Pull request by fleetwoods**
Capture SMS metrics
IssuedToGateway metric now triggered by IssuedForDelivery event
Added service tests
Auto formatting changes

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Make ScalaTest write test reports that CircleCI understands
+
+// Make ScalaTest write test reports that CirceCI understands
```
Comment (cb372) [inline build.sbt]: not sure how you introduced a typo (`CirceCI`) here 🙃

File: circle.yml [hunk 2]
```diff
@@ -25,6 +28,7 @@ deployment:
       - comms-ci-scripts/deploy_to_ecs.sh -s monitor-service PRD aws/container-definition.json
       - comms-ci-scripts/send_librato_deployment_event.sh
       - comms-ci-scripts/comment_on_last_merged_pr.sh
+      - comms-ci-scripts/notify_shipit.sh
```
Comment (cb372) [inline circle.yml]: 💪

File: src/main/scala/com/ovoenergy/comms/monitor/logging/LoggingWithMDC.scala [hunk 3]
```diff
@@ -42,7 +42,7 @@ trait LoggingWithMDC {
 
   private def log(traceToken: String, loggingFunction: () => Unit, commName: Option[String] = None) {
     try {
-      if(commName.isDefined){
+      if (commName.isDefined) {
```
Comment (cb372) [inline src/main/scala/com/ovoenergy/comms/monitor/logging/LoggingWithMDC.scala]: `commName.foreach(MDC.put("commName", _))`

**Comment by laurence-bird** [comment]
👍 👍

File: build.sbt (hunk 1 above)
Comment (fleetwoods) [inline build.sbt]: Haha, you made me Circe obsessed....copied from another project that had the changes in the line below I needed :(

**Comment by ovo-comms-circleci** [comment]
This PR was successfully deployed to production by [CircleCI build 184](https://circleci.com/gh/ovotech/comms-monitor-service/184) at 2017-04-05 15:51 BST.

**Pull request merged**